plugins/backend/backstage-plugin-argo-cd-backend/src/service/argocd.service.ts

```diff
@@ -951,24 +1002,38 @@ export class ArgoService implements ArgoServiceApi {
     return true;
   }
 
+  // @see https://cd.apps.argoproj.io/swagger-ui#operation/ApplicationService_List
   async getArgoApplicationInfo({
     argoApplicationName,
     argoInstanceName,
+    baseUrl,
+    argoToken,
   }: {
     argoApplicationName: string;
-    argoInstanceName: string;
+    argoInstanceName?: string;
```

Comment:
Any reason to make these optional if they are actually required?

Reply:
I will fix the type to be accurate.

Reply:
great catch!